server/storage/rethinkdb.go

```diff
@@ -29,7 +30,22 @@ type RDBTUFFile struct {
 
 // TableName returns the table name for the record type
 func (r RDBTUFFile) TableName() string {
-	return "tuf_files"
+	return TUFFileTableName
+}
+
+// Change defines the the fields required for an object in the changefeed
+type Change struct {
+	ID        string    `gorethink:"id,omitempty"`
```

Comment:
I'm no database expert, but is there a reason for this ID being a `string` vs. the mysql version being a `uint`?

Reply:
RethinkDB IDs are UUID strings